init/pkgstats.timer

```diff
@@ -6,3 +6,6 @@ OnCalendar=weekly
 AccuracySec=1d
 RandomizedDelaySec=7d
 Persistent=true
+
+[Install]
+WantedBy=default.target
```

Comment:
I think this should be the timers.target though (which is already symlinked by the package)